x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/persistence_directory_services_plugins_modification.json

```diff
@@ -0,0 +1,50 @@
+{
+  "author": [
+    "Elastic"
+  ],
+  "description": "Identifies the creation or modification of a DirectoryService PlugIns (dsplug) file. The DirectoryService daemonlaunches on each system boot and automatically reloads after crash. It scans and executes bundles that are located in the DirectoryServices PlugIns folder and can be abused by adversaries to maintain persistence.",
+  "false_positives": [
+    "Trusted DirectoryService PlugIns"
```

Comment:
doesn't add much to the rule, seems obvious based off the name

Reply:
I think it would be more helpful to tell the user how to identify trusted vs non-trusted. Otherwise it's like saying "some false positives for this rule could be legitimate applications." It's a given but not very helpful

Reply:
we don't have telemetry for FPs to provide specific/granular details about FPs. how to identify FPs could be lot of steps and goes beyond rule content.

Reply:
removed the false_positives section as I don't have a better alternative for now.